src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -32,7 +32,7 @@
 DEPENDENCIES = [
     'azure-cli-core',
     'azure-functions-devops-build==0.0.11',
-    'azure-mgmt-web==0.40.0',
+    'azure-mgmt-web==0.41.0',
```

Comment:
Bumping this version to 0.41.0 will need the tests to be re-run & re-recorded if needed. Started a PR for this https://github.com/Azure/azure-cli/pull/8886  do you want to wait for this PR to be merged or do you want to run & record all the tests yourself as a part of this? Let me know. Thanks!

Reply:
Thanks for letting me know about that. I see that you have already rerecorded the tests in your PR. I can wait for your PR to be checked in (looks like it should be in a day or so?). 

Please let me know once your PR is in, I can rebase and update mine.